packages/procaptcha-bundle/src/index.tsx

```diff
@@ -195,12 +195,12 @@ const implicitRender = () => {
 }
 
 // Explicit render for targeting specific elements
-export const render = (elementId: string, renderOptions: ProcaptchaRenderOptions) => {
+export const render = (element: Element, renderOptions: ProcaptchaRenderOptions) => {
     const siteKey = renderOptions.siteKey
-    const element = document.getElementById(elementId)
+    // const element = document.getElementById(elementId)
```

Comment:
Please remove this instead of commenting out

Reply:
Okay, I got it